### DISCUSSION TITLE
Is regex/string match supported for excluding/including templates from within a Directory?

### DISCUSSION THREAD

**Discussion by RogueSMG** [projectdiscovery/nuclei#4628] [Q&A] [answered]
Eg. Can I use the following in either the config file or as a flag to exclude certain matching templates from within a directory?
```
# template or template directory to exclude (comma-separated, file)
exclude-templates:
  - http/technologies/*-detect.yaml
```

**Comment by olearycrew** [comment, marked as answer]
I don't believe this is supported, no. Feel free to open an issue or PR to add this functionality. Thanks!
